python/VoxelMapper.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import os
+import nibabel as nib
+import  numpy as np
+
+for line in sys.stdin:
+	line = line.strip()
+	slice_file = os.path.join("/data/test/2dslice/" + line)
+	
+        #open nifti image
+        slice = nib.load(slice_file)
+	data = slice.get_data()
+
+        #iterate throught the voxels and assign a value of 1 to each voxel encountered
+	for row in data:
+		for key in row:
+			value = 1
+			print("%s\t%d" % (key, value))
```

Comment:
Also, note that this wouldn't behave the same as the Java implementation in case a float type is used.